index.html

```diff
@@ -0,0 +1,200 @@
+    <nav id="mobile-menu">
+      <ul class="mobile-menu-list">
+        <li class="mobile-menu-item">
+          <img id="cancel-btn" src="assets/img/cancelBtn.png" alt="Cancel">
+        </li>
+        <li class="mobile-menu-item">
+          <a href="about.html">About</a>
+        </li>
+        <li class="mobile-menu-item">
+          <a href="#program-section">Features</a>
+        </li>
+        <li class="mobile-menu-item">
+          <a href="#advisors-section">Advisors</a>
+        </li>
+        <li class="mobile-menu-item">
+          <a href="about.html#partners-section">Partners</a>
+        </li>
+        <li class="mobile-menu-item">
+          <a href="https://www.dubaiairshow.aero/news/industry-news" target="_blank" rel="noopener noreferrer">Media</a>
+        </li>
+      </ul>
+    </nav>
+    <div class="nav-bar">
+      <nav id="nav-bar1">
+        <div class="social">
+          <a><i class="fab fa-facebook-f"></i></a>
+          <a><i class="fab fa-twitter"></i></a>
+        </div>
+        <ul class="nav-bar1-list">
+          <li class="nav-bar1-item">
+            <a href="#">Arabic</a>
+          </li>
+          <li class="nav-bar1-item">
+            <a href="#">Register</a>
+          </li>
+          <li class="nav-bar1-item">
+            <a href="#">Login</a>
+          </li>
+        </ul>
+      </nav>
+      <nav id="nav-bar2">
+        <a id="logo" href="index.html"></a>
+        <ul class="nav-bar2-list">
+          <li class="nav-bar2-item">
+            <a href="about.html">About</a>
+          </li>
+          <li class="nav-bar2-item">
+            <a href="#program-section">Features</a>
+          </li>
+          <li class="nav-bar2-item">
+            <a href="#advisors-section">Advisors</a>
+          </li>
+          <li class="nav-bar2-item">
+            <a href="#partners-section">Partners</a>
+          </li>
+          <li class="nav-bar2-item">
+            <a href="https://www.dubaiairshow.aero/news/industry-news" target="_blank" rel="noopener noreferrer">Media</a>
+          </li>
+          <li id="exhibition" class="nav-bar2-item">
+            <a href="https://www.dubaiairshow.aero/floor-plan-download" target="_blank" rel="noopener noreferrer">Exhibition</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <section id="headline-section">
+    <h3 class="subtitle">The future of aerospace industry</h3>
+    <div>
+      <h1 id="headline-title">
+        DUBAI AIRSHOW
+        <br>
+        EXHIBITION
+        <br>
+        2023
+      </h1>
+    </div>
+    <p class="description">
+      A biennial air show held in Dubai and organised by Tarsus Aerospace since 1989, 
+      and it is open to business professionals and industry only.
+    </p>
+    <h3 class="schedule">2023.11.13(MON) ~ 17(FRI)</h3>
+    <p id="location">@ Dubai World Central, United Arab Emirates</p>
+  </section>
+  <section id="program-section">
```

Comment:
- [ ] [OPTIONAL] Good job so far :clap: But I think there are a few things with your design you need to work on. Please make sure that when the topic card has hovered with the mouse you have a border around it like on the screenshot below 😄  

#### Orignal design
![Template2](https://user-images.githubusercontent.com/6615394/210799135-2796a81c-2ff1-4950-9d6b-1f7687384658.jpg)

#### Your design
![Capture](https://github.com/tajemouti/first-capstone/assets/6615394/4cbe0236-9507-4caf-a965-e4aea4d5788e)